tests/rbd_mirror/test_rbd_mirror_reconfig.py

```diff
@@ -0,0 +1,79 @@
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """Configure RBD mirroring for pool based,
+    Attempt creating mirror in same cluster for
+    both pool based and image based, This should be prevented.
+    Args:
+        **kw:
+    Returns:
+        0 - if test case pass
+        1 - it test case fails
+    """
+    try:
+        log.info("Starting RBD mirroring test case")
+        config = kw.get("config")
+        mirror1, mirror2 = [
```

Comment:
Could you please use rbd_mirror_config method so that we can cover ecpool implementation part as well while automating the test case itself.